### PR TITLE
[bug 1116523] Add respons ID to session

### PR DESCRIPTION
**Pull request by aokoye**
I added a test as for the new code in fjord/feedback/views.py as well.

### CODE REVIEW

File: fjord/feedback/tests/test_views.py [hunk 1]
```diff
@@ -60,6 +60,23 @@ def test_valid_happy(self):
         # Make sure it doesn't create a context record
         eq_(models.ResponseContext.objects.count(), 0)
 
+    def test_opinion_id_in_session(self):
+        """Test that the opinion ID is in the session.
+
+        """
+        amount = models.Response.objects.count()
```
Comment (mythmon) [inline fjord/feedback/tests/test_views.py]: You probably don't need this line at all in this test.

**Comment by aokoye** [comment]
I deleted the line that Mike pointed out and also wrapped the docstring on a single line.

File: fjord/feedback/tests/test_views.py [hunk 2]
```diff
@@ -60,6 +60,21 @@ def test_valid_happy(self):
         # Make sure it doesn't create a context record
         eq_(models.ResponseContext.objects.count(), 0)
 
+    def test_opinion_id_in_session(self):
+        """Test that the opinion ID is in the session."""
+
+        url = reverse('feedback', args=(u'firefox',))
+        r = self.client.post(url, {
```
Comment (willkg) [inline fjord/feedback/tests/test_views.py]: Nit: You never use `r`, so you don't need to create it.

**Comment by willkg** [comment]
One minor nit, but everything else looks good!

**Comment by aokoye** [comment]
I fixed the nit!

**Comment by willkg** [comment]
Awesome! Can you squash these into one commit? Then it'll be ready to go.

**Comment by aokoye** [comment]
Commits have been squashed!

**Comment by willkg** [comment]
Landed in 28ad328 [bug 1116523] Add response ID to session

Good job!

**Pull request merged**